frontend/src/clients/PipelineToolClient.ts

```diff
@@ -25,7 +25,7 @@ export class PipelineToolClient extends HttpClient {
         throw new BadRequestException(params.type, 'verify failed')
       }
       if (code === 500) {
-        throw new BadServerException(params.type, 'verify failed')
+        throw new InternalServerException(params.type, 'verify failed')
```

Comment:
'verify failed' not make scense
PLZ use Internal server error

Reply:
> 'verify failed' not make scense PLZ use Internal server error

it should be resolved by ADM-350